cmd/subctl/cloud.go

```diff
@@ -64,9 +64,8 @@ func init() {
 		port.NATTDiscovery, "NAT discovery port")
 	cloudPrepareCmd.PersistentFlags().Uint16Var(&cloudPorts.Vxlan, "vxlan-port", port.IntraClusterVxLAN, "Internal VXLAN port")
 
-	cloudPorts.Metrics = append(cloudPorts.Metrics, 8080, 8081)
-
 	cloudPrepareCmd.PersistentFlags().Var(&uint16Slice{value: &cloudPorts.Metrics}, "metrics-ports", "Metrics ports")
+	_ = cloudPrepareCmd.PersistentFlags().MarkDeprecated("metrics-ports", "Metrics ports no longer required")
```

Comment:
Let’s make it clear that `--metrics-ports` has no actual effect any more, it’s not just that it’s no longer required:

```suggestion
	_ = cloudPrepareCmd.PersistentFlags().MarkDeprecated("metrics-ports", "Metrics ports are no longer used, this flag is ignored")
```